### DISCUSSION TITLE
Button to delete all CodeSpaces

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#68476] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I  would like to create a button to delete all CodeSpaces at once. It is difficult to delete them one by one.

**Comment by mehulrajdev** [comment, marked as answer]
GitHub didn't have a built-in way to delete all CodeSpaces at once. You typically had to delete them one by one manually.
However, you can use the GitHub API and some scripting to automate this process.
